src/platforms/java/migration.mdx

```diff
@@ -0,0 +1,110 @@
+---
+title: Migration Guide
+sidebar_order: 1000
+---
+
+## Migrating from `io.sentry:sentry` `1.x` to `io.sentrt:sentry` `3.x`
+
+
+We've updated the API to follow the [Unified API](https://develop.sentry.dev/sdk/unified-api/) more closely. It's a completely new code base, that was written to support a new Android SDK.
+
+### API Changes
+
+**Set tag**
+
+_Old_:
```

Comment:
Entirely your choice, but this may age better if it's (I'll suggest here; it should be modified throughout the file)
```suggestion
_Previous_:
```